### PR TITLE
Bump Fast DDS and Fast CDR

### PR DESCRIPTION
**Pull request by pablogs9**
Signed-off-by: Pablo Garrido <pablogs9@gmail.com>

### CODE REVIEW

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Linux/564/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Windows/565/)

File: CMakeLists.txt [hunk 1]
```diff
@@ -95,8 +95,8 @@ if(UAGENT_FAST_PROFILE)
     if(UAGENT_USE_SYSTEM_FASTDDS)
         set(_fastdds_version 2)
     else()
-        set(_fastdds_version 2.4.1)
-        set(_fastdds_tag v2.4.1)
+        set(_fastdds_version 2.8.0)
+        set(_fastdds_tag v2.8.x)
```
Comment (pablogs9) [inline CMakeLists.txt:99]:
```suggestion
        set(_fastdds_tag 2.8.x)
```

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Linux/565/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Windows/566/)

**Review by Acuadros95 — APPROVED** [review]

**Comment by pablogs9** [comment]
Integration tests here: https://github.com/eProsima/Micro-XRCE-DDS/pull/148

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Linux/566/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Windows/567/)

**Comment by pablogs9** [comment]
CI ok: https://github.com/eProsima/Micro-XRCE-DDS/pull/148

**Pull request merged**